tests/foreman/cli/test_oscap_tailoringfiles.py

```diff
@@ -224,36 +237,24 @@ def test_positive_download_tailoring_file(self):
             1.Create valid tailoring file with valid name
             2.Execute "tailoring-file" command with "download" as sub-command
 
-        :CaseAutomation: notautomated
+        :CaseAutomation: automated
 
         :expectedresults: The tailoring file should be downloaded
 
         :CaseImportance: Critical
         """
+        name = gen_string('alphanumeric')
+        tailoring_file = make_tailoringfile({
+            'name': name,
+            'scap-file': '/tmp/{0}'.format(self.file_name)})
+        self.assertEqual(tailoring_file['name'], name)
+        result = TailoringFiles.download_tailoring_file({
+            'name': name,
+            'path': '/tmp/'
+        })
+        self.assertIn('/tmp/{0}.xml'.format(name), result[0])
```

Comment:
perhaps we should ensure that file also exists in the /tmp folder by executing cmd "find" or "ls"
```python
file_path = '/tmp/{0}.xml'.format(name)
result = ssh.command('find {0}'.format(file_path))
self.assertEqual(result.return_code, 0)
self.assert(file_path, result.stdout)
```